### ISSUE TITLE
Missing flux factor in 2016arXiv161003751S/tev-000030-lc.ecsv

### ISSUE THREAD

**Issue by cdeil** [gammapy/gamma-cat#114]
@GernotMaier - There's a bug with the data entry for this lightcurve file:
https://github.com/gammapy/gamma-cat/blob/master/input/data/2016/2016arXiv161003751S/tev-000030-lc.ecsv

It currently has:
```
# - {name: flux, unit: cm-2 s-1, datatype: float64}
# - {name: flux_err, unit: cm-2 s-1, datatype: float64}
e_min time_min time_max time flux flux_err
0.350 54830 54834 54832.0 1.3093 0.4514
```

There's a factor of 1e-10 or 1e-11 or 1e-12 missing for the `flux` and `flux_err` column, no?

Could you please add the correct factor?
Note that if you want, you can add it as a scale factor with the unit and that will be processed correctly, i.e. for example if it's `1e-12`:
```
# - {name: flux, unit: 1e-12 cm-2 s-1, datatype: float64}
```

**Comment by GernotMaier** [comment]
Fixed this (was 1e-12).

**Comment by cdeil** [comment]
For reference: this was fixed in 0ef4d9dee7248d00339def84432f6561b4c2958e

**Comment by cdeil** [comment]
@GernotMaier - Thanks!
